contracts/button/lib.rs

```diff
@@ -58,6 +58,19 @@ pub mod button_game {
         score: Balance,
     }
 
+    /// Event emitted when a reward token is minted to a players account
+    ///
+    /// Could be a regular player or the Pressiah
+    #[ink(event)]
+    #[derive(Debug)]
+    pub struct RewardMinted {
+        when: BlockNumber,
```

Comment:
> everything looks okay to me, thus approve, but I kinda miss the point of this scaling by 10^12 - afaik, these 12 decimal places are relevant only to the chain's _native_ token
> 
> is this because tooling was displaying these rewards in an 'unpleasant' way, like it would do for low amounts of AZERO?
> 
> also, could this change introduce some numerical problems to rewards marketplace?

Because rewards are paid out proportionally to the time-to-deadline (or time elapsed) and games live for ~8600 blocks or less, they are always in pico/nano units. Wallets aren't best suited to displaying those and it might generally be poorly understood that you got e.g 1e-8 of a reward. 

@obrok recently made some fixes to the linear price drop calculations so this should not have an impact.